src/Extension/FluentExtension.php

```diff
@@ -1068,7 +1068,7 @@ public function getLocaleInstances(): array
     {
         $locales = [];
         foreach ($this->owner->Locales() as $info) {
-            if ($info->IsDraft()) {
+            if (method_exists($info, 'IsDraft') && $info->IsDraft()) {
```

Comment:
if method_exists returns false, what is $info? Is it not an `RecordLocale` instance ?